### ISSUE TITLE
Loading WBP reconstruction state file

### ISSUE THREAD

**Issue by ercius** [OpenChemistry/tomviz#1504]
WBP works correctly first time. Then save/reload state file.

Error thrown:
>   File "Reconstruct (Back Projection)", line 32, in transform_scalars
> TypeError: 'float' object cannot be interpreted as an integer
> Failed to execute the script.

line 32:
```python 
recon = np.empty([Nslice, Nrecon, Nrecon], dtype=float, order='F')
```
Ignore `dtype=float`. This is not the issue!

This occurs because Nslice is being passed as a float now. Need to change JSON input from user interface to integer.

**Comment by ercius** [comment]
This might be connected to #1496

**Comment by mathturtle** [comment]
It looks like this was fixed by #1515.

**Comment by cryos** [comment]
Fixed
